Userland/Libraries/LibM/math.cpp

```diff
@@ -1116,6 +1116,21 @@ float fminf(float x, float y) NOEXCEPT
     return x < y ? x : y;
 }
 
+long double fmal(long double x, long double y, long double z) NOEXCEPT
```

Comment:
Spec comments for these? https://pubs.opengroup.org/onlinepubs/9699919799/functions/fmal.html